Taskfile.yaml

```diff
@@ -116,6 +106,16 @@ tasks:
         docker compose run --rm mongosh mongosh mongodb://host.docker.internal:27017/
         --verbose --eval 'disableTelemetry()' --shell
 
+  init-repl:
+    desc: "Initializes a new replica set"
+    cmds:
+      - >
+        docker compose run --rm mongosh mongosh
+        'mongodb://host.docker.internal:27017/'
+        --eval 'db.getSiblingDB("admin").runCommand({replSetInitiate: {_id: "mongodb-rs", members:[{_id: 0, host: "localhost"}]}});'
```

Comment:
Can we use `rs.initiate` as we do in FerretDB? See https://github.com/FerretDB/FerretDB/blob/2664b9a5dd570d8157fc690a018934171908575c/cmd/envtool/envtool.go#L184

Reply:
I needed to get a connection to the admin DB, but it now seems to work using the shell method. Strange.